CooperationHullMainSite/wwwroot/css/site.css

```diff
@@ -269,7 +269,6 @@ h1 {
 
 
 .main-content {
-    background-color: var(--city-orange);
```

Comment:
To ensure that we can overlay the call out block on sibling section's backgrounds, we no longer set a global background colour, and instead set one per section